core/domain/email_manager.py

```diff
@@ -1159,6 +1270,328 @@ def send_mail_to_notify_users_to_review(user_id, category):
             email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
 
 
+def _create_html_for_reviewable_suggestion_email_info(
+        reviewable_suggestion_email_info):
+    """Creates the html for the given reviewable_suggestion_email_info. This
+    html content is used to provide information about a suggestion in an
+    email.
+
+    Args:
+        reviewable_suggestion_email_info: ReviewableSuggestionEmailInfo. The
+            information about the suggestion that will be used to form the
+            html for the email. This includes the suggestion type, language,
+            content and review submission date.
+
+    Returns:
+        str. A string containing the html that represents the suggestion
+        information.
+    """
+    # Get the language of the suggestion.
+    language = utils.get_supported_audio_language_description(
+        reviewable_suggestion_email_info.language_code)
+    # Calculate how long the suggestion has been waiting for review.
+    suggestion_review_wait_time = (
+        datetime.datetime.utcnow() - (
+            reviewable_suggestion_email_info.submission_datetime))
+    # Get a string composed of the largest time unit that has a
+    # value, followed by that time unit. For example, if the
+    # suggestion had been waiting for review for 5 days and 2 hours,
+    # '5 days' would be returned. This is more user friendly since a
+    # high level of precision is not needed.
+    human_readable_review_wait_time = (
+        utils.create_string_from_largest_unit_in_timedelta(
+            suggestion_review_wait_time))
+    values_to_populate_suggestion_template_dict = {
+        'language': language,
+        'review_wait_time': human_readable_review_wait_time,
+        'suggestion_content': (
+            reviewable_suggestion_email_info.suggestion_content)
+    }
+    get_values_to_populate_suggestion_template = (
+        HTML_FOR_REVIEWABLE_SUGGESTION_EMAIL_INFO[
+            'suggestion_template_values_getter_functions'][
+                reviewable_suggestion_email_info.suggestion_type])
+    suggestion_template = (
+        HTML_FOR_REVIEWABLE_SUGGESTION_EMAIL_INFO[
+            'suggestion_template'][
+                reviewable_suggestion_email_info.suggestion_type])
+    return suggestion_template % (
+        get_values_to_populate_suggestion_template(
+            values_to_populate_suggestion_template_dict))
+
+
+def send_mail_to_notify_admins_suggestions_waiting_long(
```

Comment:
Similarly, if anyone has any better naming ideas for this method that'd be great. I thought of alternatives but they were too long.

Reply:
Following the format of other functions in email_manager, I think you can just do something like: send_stale_suggestions_notification_email